website/docs/r/wafv2_web_acl_logging_configuration.html.markdown

```diff
@@ -38,12 +38,14 @@ The following arguments are supported:
 
 The `redacted_fields` block supports the following arguments:
 
-* `all_query_arguments` - (Optional) Inspect all query arguments.
-* `body` - (Optional) Inspect the request body, which immediately follows the request headers.
+~> **NOTE:** Only one of `method`, `query_string`, `single_header` or `uri_path` can be specified.
+
+* `all_query_arguments` - (Optional, **DEPRECATED**) Inspect all query arguments.
+* `body` - (Optional, **DEPRECATED**) Inspect the request body, which immediately follows the request headers.
 * `method` - (Optional) Inspect the HTTP method. The method indicates the type of operation that the request is asking the origin to perform.
 * `query_string` - (Optional) Inspect the query string. This is the part of a URL that appears after a `?` character, if any.
 * `single_header` - (Optional) Inspect a single header. See [Single Header](#single-header) below for details.
-* `single_query_argument` - (Optional) Inspect a single query argument. See [Single Query Argument](#single-query-argument) below for details.
+* `single_query_argument` - (Optional, **DEPRECATED**) Inspect a single query argument. See [Single Query Argument](#single-query-argument) below for details.
 * `uri_path` - (Optional) Inspect the request URI path. This is the part of a web request that identifies a resource, for example, `/images/daily-ad.jpg`.
```

Comment:
The word _inspect_ seems wrong here. It was probably copied from the `field_to_match` in the `aws_wafv2_web_acl` resource. But here we are not matching a field but redacting it. My suggestion would be to replace _Inspect_ by _Redact_.

The same applies to the _Single Header_ section.

Reply:
Agreed, thanks for commenting here! The definition of these params are definitely context-specific. I think they should all probably be updated to use `redact` or something along the lines of `hide from logs` . will update!

Reply:
changes added in c7429ef :)